plugins/arDominionB5Plugin/scss/_layout.scss

```diff
@@ -213,3 +213,10 @@ body > iframe {
 #treeview-search .alert {
   border-color: $border-color !important;
 }
+
+// Repository logo with an approximate shape of a square.
```

Comment:
I think the square shape still depends on the image aspect ratio.

Reply:
That's right, it's an approximation.